lib/plausible_web/live/verification.ex

```diff
@@ -45,28 +47,65 @@ defmodule PlausibleWeb.Live.Verification do
         attempts: 0
       )
 
-    if connected?(socket) do
-      launch_delayed(socket)
+    on_ee do
+      if connected?(socket) do
+        launch_delayed(socket)
+      end
+    end
+
+    on_ee do
+      {:ok, socket}
+    else
+      # on CE we skip the verification process and instead,
+      # we just wait for the first pageview to be recorded
+      socket =
+        if has_pageviews? do
+          redirect_to_stats(socket)
+        else
+          schedule_pageviews_check(socket)
+        end
+
+      {:ok, socket}
     end
+  end
 
-    {:ok, socket}
+  on_ee do
+    def render(assigns) do
+      ~H"""
+      <PlausibleWeb.Components.FlowProgress.render flow={@flow} current_step="Verify installation" />
+
+      <.live_component
+        module={@component}
+        installation_type={@installation_type}
+        domain={@domain}
+        id="verification-standalone"
+        attempts={@attempts}
+        flow={@flow}
+        awaiting_first_pageview?={not @has_pageviews?}
+        super_admin?={@super_admin?}
+      />
+      """
+    end
+  else
+    def render(assigns) do
+      ~H"""
+      <PlausibleWeb.Components.FlowProgress.render flow={@flow} current_step="Verify installation" />
+      <.awaiting_pageviews />
+      """
+    end
   end
 
-  def render(assigns) do
-    ~H"""
-    <PlausibleWeb.Components.FlowProgress.render flow={@flow} current_step="Verify installation" />
-
-    <.live_component
-      module={@component}
-      installation_type={@installation_type}
-      domain={@domain}
-      id="verification-standalone"
-      attempts={@attempts}
-      flow={@flow}
-      awaiting_first_pageview?={not @has_pageviews?}
-      super_admin?={@super_admin?}
-    />
-    """
+  on_ce do
+    defp awaiting_pageviews(assigns) do
+      ~H"""
+      <PlausibleWeb.Components.Generic.focus_box>
+        <div class="flex items-center">
+          <div class="block pulsating-circle"></div>
+          <p class="ml-8">Awaiting a page view ...</p>
```

Comment:
```suggestion
          <p class="ml-8">Awaiting your first pageview...</p>
```

Minor, to make it similar to what we communicate elsewhere

Reply:
I had it like this at first, but then changed to "a page view" to make it somewhat relevant in the flow=review.

Reply:
I guess the review flow is sort-of broken anyway, so we can call this check the "first pageview" there as well. Thank you!

Reply:
I think in the future CE might check not just the pageview count, but actually wait for a pageview to arrive after the check starts.